.gitignore

```diff
@@ -71,28 +59,21 @@ rsconnect/
 6_temp_coop_fetch/downloads/*.csv
 7_drivers_munge/mendota_out/*
 7_drivers_munge/sparkling_out/*
-
 6_drivers/in/*.csv
 !6_drivers/in/*.rds
-
 # tmp wqp indicators
 6_temp_wqp_fetch/tmp/*
-
 # tmp coop munged files/indicators
 7a_temp_coop_munge/tmp/*
 build/status/N2FfdGVtcF9jb29wX211bmdlL3RtcC9*
 !7a_temp_coop_munge/tmp/parser_files.yml
-
 # tmp wqp munge indicators
 7a_temp_wqp_munge/tmp/*
-
-
 # ignore bathy downloads
 3_params_fetch/in/hypsos_m/*
-
 # ignore html files
 *.html
-
 # ignore source metdata csv
 7b_temp_merge/in/source_metadata.csv
 7b_temp_merge/out/source_metadata_for_release.csv
+sentinel_lake_names.csv
```

Comment:
Changes in this doc are not relevant to this PR. Looks like some line ending issues + an unrelated change?

Reply:
It is unrelated change. I added the file that was setting in the my local project to the .gitignore file.